docs/src/_concepts/plugins.md

```diff
@@ -688,16 +696,26 @@ export TAP_GITLAB__VARS='{"schema": "{{ env_var(''DBT_SOURCE_SCHEMA'') }}"}'
 ```
 ### Orchestrators
 
+<div class="notification is-warning">
+  <p> Orchestrator plugins are transitioning over to being called Utilities. The new approach is to group all non-EL plugins under the `utility` plugin type. </p>
+  <p> The orchestrator plugin type is still supported for now but will eventually be phased out as utilities take over.</p>
+</div>
+
 Orchestrators are [pip packages](https://pip.pypa.io/en/stable/) responsible for [orchestrating](/guide/orchestration) a project's [scheduled pipelines](/reference/command-line-interface#schedule).
 
-Meltano supports [Apache Airflow](https://airflow.apache.org/) out of the box, but can be used with any tool capable of reading the output of [`meltano schedule list --format=json`](/reference/command-line-interface#schedule) and executing each pipeline's [`meltano elt`](/reference/command-line-interface#elt) command on a schedule.
+Meltano supports [Apache Airflow](https://airflow.apache.org/) out of the box, but can be used with any tool capable of reading the output of [`meltano schedule list --format=json`](/reference/command-line-interface#schedule) and executing each pipeline's [`meltano run`](/reference/command-line-interface#run) command on a schedule.
 
 When the `airflow` orchestrator is added to your project using [`meltano add`](/reference/command-line-interface#add),
 its related [file bundle](#file-bundles) will automatically be added as well.
 
 ### Transformers
 
-Transformers are [pip packages](https://pip.pypa.io/en/stable/) used by [`meltano elt`](/reference/command-line-interface#elt) as part of [data transformation](/guide/transformation).
+<div class="notification is-warning">
```

Comment:
Thoughts on included the same warning for files bundles? Long-term I think we'd want to have those all EDK-based even if all it does is inject some files.

Reply:
@tayloramurphy I didnt know what our long term vision was for those so I just left them. I'd have to think about that a little more 🤔 .

Reply:
That's fine for now. Can you open an internal discussion on that when you get a chance and we can get alignment on it?

Reply:
@tayloramurphy I created https://github.com/meltano/internal-general/discussions/469 to discuss it.